build/index.js

```diff
@@ -9,5 +9,13 @@ const template = require.resolve('govuk_template_mustache/views/layouts/govuk_te
 
 const govukTemplate = fs.readFileSync(template, { encoding: 'utf-8' });
 const compiledTemplate = Hogan.compile(govukTemplate).render(govukConfig);
+const parsedTemplate = addNonceValueAttributeToInlineScripts(compiledTemplate);
 const output = path.resolve(__dirname, '../govuk_template.html');
-fs.writeFileSync(output, compiledTemplate, { encoding: 'utf-8' });
+
+fs.writeFileSync(output, parsedTemplate, { encoding: 'utf-8' });
+
+function addNonceValueAttributeToInlineScripts(compiledTemplate) {
+  const scriptNonceValue = `{{#nonce}}nonce="{{nonce}}"{{/nonce}}`
+  return compiledTemplate.replace(/(?<tag><script)(?<content>(?![^>]+src).*?)>/g,
+    `$<tag>$<content> ${scriptNonceValue}>`);
```

Comment:
`<content> ` is deprecated and shouldn't be used on production sites: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content

Reply:
This was functioning as a capture group tag in the regex, but I have renamed it for clarity.